src/ethereum_test_forks/base_fork.py

```diff
@@ -258,6 +260,14 @@ def solc_name(cls, block_number: int = 0, timestamp: int = 0) -> str:
         """
         pass
 
+    @classmethod
+    @abstractmethod
+    def solc_min_version(cls, block_number: int = 0, timestamp: int = 0) -> Version:
```

Comment:
Couldn't we omit the `block_number` and `timestamp` args here?

Reply:
The problem is on the transition forks generator, if it's an abstract method, then it will try to perform some actions on it, and it needs the parameters to be `block_number` and `timestamp` precisely.

See here: https://github.com/ethereum/execution-spec-tests/blob/77f887a7d3d1ea446d282c570fb394e276e8d5f3/src/ethereum_test_forks/transition_base_fork.py#L71-L77

I think we could get away with not marking this method as abstract maybe?

Reply:
Nice!